edx_proctoring/tests/test_student_view.py

```diff
@@ -764,6 +764,14 @@ def test_get_studentview_new_onboarding(self):
         rendered_response = self.render_onboarding_exam()
         self.assertIn('Proctoring onboarding exam', rendered_response)
 
+    def test_get_onboarding_no_perm(self):
+        """
+        Test that onboarding exams are gated by the same permission as proctored exams
```

Comment:
Comment's misleading; the permissions can easily diverge with this test staying green.